packages/data/tests/unit/models/metadata/dimension-test.ts

```diff
@@ -125,12 +129,14 @@ module('Unit | Metadata Model | Dimension', function (hooks) {
 
     assert.deepEqual(Dimension.primaryKeyFieldName, 'id', 'primaryKeyFieldName returned `id` as the primary key field');
 
-    let nonId = DimensionMetadataModel.create({
+    //@ts-expect-error
```

Comment:
Why do we have errors here? Missing fields?

Reply:
Yea now that `.create` expects the correct payload to be passed in this gives a type error because of the missing fields